### ISSUE TITLE
[Bug]: The values of the reported metrics `QueryNodeSearchGroupNQ` and `QueryNodeSearchGroupTopK` are incorrect

### ISSUE THREAD

**Issue by wangting0128** [milvus-io/milvus#24915]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 
- Deployment mode(standalone or cluster):
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

![image](https://github.com/milvus-io/milvus/assets/26307815/a18ffc90-a353-4c1e-8c17-14d54e03d009)

### Expected Behavior

- **Statistical nq total value**
- **Report the maximum topk value**

**Comment by yah01** [comment]
/assign @wangting0128 
fixed with #24916

**Comment by wangting0128** [comment]
verify done
